resources/authgear/templates/en/translation.json

```diff
@@ -76,6 +76,7 @@
   "oauth-branding-facebook": "Login with Facebook",
   "oauth-branding-linkedin": "Sign in with LinkedIn",
   "oauth-branding-azureadv2": "Sign in with Microsoft",
+  "oauth-branding-wechat": "Sign in with WeChat",
```

Comment:
The wording should be `Login with WeChat`. See https://developers.weixin.qq.com/doc/oplatform/en/Downloads/Design_Resource.html

Reply:
Seems the link only shows the Chinese version button? I updated the wording anyway.